Description: Correct pod NAME issue in LedgerSMB/OE.pm
 Is missing a pod NAME entry altogether.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472610&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/OE.pm
+++ b/LedgerSMB/OE.pm
@@ -1,3 +1,11 @@
+=pod
+
+=head1 NAME
+
+LedgerSMB::OE - Order Entry
+
+=cut
+
 #=====================================================================
 # LedgerSMB
 # Small Medium Business Accounting software
